core/domain/exp_domain.py

```diff
@@ -1611,6 +1611,57 @@ def create_default_state(
                 default_dest_state_name))
 
 
+class ExplorationVersionsDiff(object):
+    """Domain object for the difference between two versions of an Oppia
+    exploration.
+
+    Attributes:
+        states_added: list(str). The states added to the exploration from
+            prev_exp_version to current_exp_version.
+        states_deleted: list(str). The states deleted from the exploration from
+            prev_exp_version to current_exp_version.
+        states_renamed: dict. Dictionary mapping state names of
+            current_exp_version to the state names of prev_exp_version.
+    """
+
+    def __init__(self, change_list):
+        """Constructs an ExplorationVersionsDiff domain object.
+
+        Args:
+            change_list: list(dict). A list of all of the commit cmds from the
+                old_stats_model up to the next version.
```

Comment:
old_stats_model? Do you mean the old version of the exploration?

Reply:
Done.